### DISCUSSION TITLE
Release circe-yaml v0.14.3

### DISCUSSION THREAD

**Discussion by jeffmay** [circe/circe-yaml#364] [General]
I am tagging everyone here who has helped to create v0.14.3-RC tagged versions. Do you feel confident in releasing v0.14.3? Is there any concerns about backwards compatibility we should be aware of before releasing this version?

@lucaviolanti 
@sideeffffect 

If you both have had a chance to verify your features and feel confident enough to release, I want to get this version tagged before releasing [v0.15.0](https://github.com/circe/circe-yaml/discussions/363).

Thanks in advance!

**Comment by sideeffffect** [comment]
I think we should be fine, we have MiMa checks set up, don't we?
Otherwise yes, I'm fine with making a new release :+1: 
Thanks for taking charge of this :bow:

**Reply by laughedelic** [reply]
Hi! Could you cut a release for this patch?

**Reply by sideeffffect** [reply]
Are the RC versions no good for you?
https://github.com/circe/circe-yaml/releases

**Reply by laughedelic** [reply]
We added dependency on 0.15.0-RC1 to resolve the vulnerability alert, but we would like to have a _stable_ dependency. RC suffix _usually_ indicates that the release is considered not stable enough. It's just about keeping dependencies clean.

**Comment by jeffmay** [comment]
These changes were released in https://github.com/circe/circe-yaml/releases/tag/v0.15.0
